app/src/androidTest/java/org/mozilla/focus/screenshots/TVScreenshots.java

```diff
@@ -170,7 +170,7 @@ public void settingsView() {
         // capture a screenshot of the clear data dialog
         clearButton.perform(click());
 
-        onView(allOf(withText(R.string.settings_cookies_dialog_content), isDisplayed())).inRoot(isDialog());
```

Comment:
Should this older string be removed? (or is it waiting on a string import to be removed?)

Reply:
Not sure why github is doing that. I just changed `R.string.settings_cookies_dialog_content` to `R.string.settings_cookies_dialog_content2`

Reply:
@psymoon I meant that R.string.settings_cookies_dialog_content appears to exist for non-enUS locales so I was wondering if we needed to do some clean up (but I think these should be cleared with string imports if the en-US string is removed...)